src/test/java/TaskTest.java

```diff
@@ -0,0 +1,18 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
could you please double check if static import needs to be imported before non-static, i think so but i am not 100% sure haha